src/test/scala/com/snowflake/snowpark_test/UdxOpenTelemetrySuite.scala

```diff
@@ -0,0 +1,100 @@
+package com.snowflake.snowpark_test
+
+import com.snowflake.snowpark.types.{IntegerType, StructField, StructType}
+import com.snowflake.snowpark.udtf.UDTF0
+import com.snowflake.snowpark.{OpenTelemetryEnabled, Row, Session, TestUtils, functions}
+
+class UdxOpenTelemetrySuite extends OpenTelemetryEnabled {
+  override def beforeAll: Unit = {
+    super.beforeAll
+    if (!isStoredProc(session)) {
+      TestUtils.addDepsToClassPath(session)
+    }
+  }
+
+  test("udf") {
+    val className = "snow.snowpark.UDFRegistration"
+    val func = () => 100
+    session.udf.registerTemporary(func)
```

Comment:
Is this the only way to create udf in java?

Reply:
There are for ways to create UDF.
```scala
session.udf.registerTemporary(func)
session.udf.registerTemporary(udfName, func)
functions.udf(func)
session.udf.registerPermanent(udfName2, func, stageName)
```
I included all of them in the tests